apps/sms/index.html

```diff
@@ -134,7 +134,7 @@
   <body role="application" class="skin-comms theme-communications">
     <iframe src="/views/shared/shim_host.html" class="shim-host" hidden></iframe>
     <article id="main-wrapper" class="wrapper">
-      <section class="panel panel-InboxView" role="region" aria-hidden="true">
+      <section class="panel panel-InboxView panel-hidden" role="region" aria-hidden="true">
```

Comment:
Yeah, I see, I just was wondering how much exactly this change decreases visuallyLoaded time comparing to the rest of the changes :)